pytest_selenium/pytest_selenium.py

```diff
@@ -207,59 +200,32 @@ def pytest_addoption(parser):
                           '"name:value".')
-    group._addoption('--chromepath',
+    group._addoption('--driver-path',
                      action='store',
-                     dest='chrome_path',
+                     dest='driver_path',
                      metavar='path',
-                     help='path to the google chrome driver executable.')
-    group._addoption('--firefoxpath',
-                     action='store',
-                     dest='firefox_path',
-                     metavar='path',
-                     help='path to the target firefox binary.')
-    group._addoption('--firefoxpref',
+                     help='path to the driver.')
+    group._addoption('--firefox-pref',
                      action='append',
                      dest='firefox_preferences',
                      metavar='str',
                      help='firefox preference name and value to set in format '
                           '"name:value".')
-    group._addoption('--profilepath',
+    group._addoption('--firefox-profile',
                      action='store',
                      dest='profile_path',
                      metavar='str',
                      help='path to the firefox profile to use.')
-    group._addoption('--extension',
+    group._addoption('--browser_extension',
                      action='append',
                      dest='extension_paths',
                      metavar='str',
                      help='path to browser extension to install.')
-    group._addoption('--chromeopts',
-                     action='store',
-                     dest='chrome_options',
-                     metavar='str',
-                     help='json string of google chrome options to set.')
-    group._addoption('--operapath',
-                     action='store',
-                     dest='opera_path',
-                     metavar='path',
-                     help='path to the opera driver.')
-    group._addoption('--browsername',
-                     action='store',
-                     dest='browser_name',
-                     default=os.environ.get('SELENIUM_BROWSER'),
-                     metavar='str',
-                     help='target browser name (default: %default).')
-    group._addoption('--browserver',
-                     action='store',
-                     dest='browser_version',
-                     default=os.environ.get('SELENIUM_VERSION'),
-                     metavar='str',
-                     help='target browser version (default: %default).')
-    group._addoption('--platform',
-                     action='store',
-                     default=os.environ.get('SELENIUM_PLATFORM'),
+    group._addoption('--chrome-option',
+                     action='append',
+                     dest='chrome_option',
                      metavar='str',
-                     help='target platform (default: %default).')
+                     help='google chrome option to set.')
```

Comment:
I'd be okay with you dropping this out of the scope of this issue and raising a separate issue to consider how we should handle driver options and arguments. Ideally we'd avoid too many browser specific options.